jax/interpreters/xla.py

```diff
@@ -953,7 +979,7 @@ def _xla_call_translation_rule(c, axis_env, in_nodes, name_stack, backend, name,
   subc = xb.make_computation_builder(f"jit_{name}")
   args = [xb.parameter(subc, i, c.get_shape(n)) for i, n in enumerate(in_nodes)]
   out_nodes = jaxpr_subcomp(subc, call_jaxpr, backend, axis_env, (),
-                            extend_name_stack(name_stack, wrap_name(name, 'jit')), *args)
+                            name_stack.extend(xla_call_p, name), *args)
```

Comment:
Here I think that this will result in a XLA op_name that includes `xla_call(foo)`, which I think is less familiar to our users than `jit(foo)`. Maybe we can change the name of the xla_call primitive?

Reply:
I am open to that, but I think we should do it in a subsequent change.